### PR TITLE
Update web servers integration documentation

### PR DESCRIPTION
**Pull request by kkujawa-sumo**
## Purpose of this pull request

Update web servers integration documentation according to changes in Helm Chart v3

Issue number:   https://jira.kumoroku.com/jira/browse/SUMO-206934

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/integrations/web-servers/apache-tomcat.md [hunk 1]
```diff
@@ -68,21 +68,26 @@ If you're using Apache Tomcat in a non-Kubernetes environment, create the fields
...
+The first service in the pipeline is Telegraf. Telegraf collects metrics from Apache Tomcat. Note that we’re running Telegraf in each pod we want to collect metrics from as a sidecar deployment, for example, Telegraf runs in the same pod as the containers it monitors. Telegraf uses the Apache Tomcat and Jolokia2 input plugin to obtain metrics. For simplicity, the diagram doesn’t show the input plugins.
+The injection of the Telegraf sidecar container is done by the Telegraf Operator.
+Prometheus pulls metrics from Telegraf and sends them to [Sumo Logic Distribution for OpenTelemetry Collector](https://github.com/SumoLogic/sumologic-otel-collector) which enriches metadata and sends metrics to Sumo Logic.
```
Comment (kimsauce) [inline docs/integrations/web-servers/apache-tomcat.md:78]:
```suggestion
Prometheus pulls metrics from Telegraf and sends them to [Sumo Logic Distribution for OpenTelemetry Collector](https://github.com/SumoLogic/sumologic-otel-collector), which enriches metadata and sends metrics to Sumo Logic.
```

File: docs/integrations/web-servers/haproxy.md [hunk 2]
```diff
@@ -126,11 +126,15 @@ Sumo Logic supports collection of logs and metrics data from HAProxy in both Kub
...
+The first service in the pipeline is Telegraf. Telegraf collects metrics from HAProxy. Note that we’re running Telegraf in each pod we want to collect metrics from as a sidecar deployment for example, Telegraf runs in the same pod as the containers it monitors. Telegraf uses the HAProxy input plugin to obtain metrics. For simplicity, the diagram doesn’t show the input plugins.
+The injection of the Telegraf sidecar container is done by the Telegraf Operator.
+Prometheus pulls metrics from Telegraf and sends them to [Sumo Logic Distribution for OpenTelemetry Collector](https://github.com/SumoLogic/sumologic-otel-collector) which enriches metadata and sends metrics to Sumo Logic.
```
Comment (kimsauce) [inline docs/integrations/web-servers/haproxy.md:135]:
```suggestion
Prometheus pulls metrics from Telegraf and sends them to [Sumo Logic Distribution for OpenTelemetry Collector](https://github.com/SumoLogic/sumologic-otel-collector), which enriches metadata and sends metrics to Sumo Logic.
```

File: docs/integrations/web-servers/nginx-ingress.md [hunk 3]
```diff
@@ -57,7 +57,7 @@ It’s assumed that you are using the latest helm chart version if not please up
...
+2. Ensure you have deployed the [Sumologic-Kubernetes-Collection](https://github.com/SumoLogic/sumologic-kubernetes-collection), to send the logs and metrics to Sumologic. For more information on deploying Sumologic-Kubernetes-Collection, [visit](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/main/docs/installation.md) here. Once deployed, logs will automatically be picked up and sent by default. Prometheus will scrape the Nginx Ingress pods, based on the annotations set in Step 1, for the metrics. Logs and Metrics will automatically be sent to the respective [Sumo Logic Distribution for OpenTelemetry Collector](https://github.com/SumoLogic/sumologic-otel-collector) instances which consistently tag your logs and metrics, then forward them to your Sumo Logic org.
```
Comment (kimsauce) [inline docs/integrations/web-servers/nginx-ingress.md:60]:
```suggestion
2. Ensure you have deployed the [Sumologic-Kubernetes-Collection](https://github.com/SumoLogic/sumologic-kubernetes-collection), to send the logs and metrics to Sumologic. For more information on deploying Sumologic-Kubernetes-Collection, [visit](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/main/docs/installation.md) here. Once deployed, logs will automatically be picked up and sent by default. Prometheus will scrape the Nginx Ingress pods, based on the annotations set in Step 1, for the metrics. Logs and Metrics will automatically be sent to the respective [Sumo Logic Distribution for OpenTelemetry Collector](https://github.com/SumoLogic/sumologic-otel-collector) instances, which consistently tag your logs and metrics, then forward them to your Sumo Logic org.
```

File: docs/integrations/web-servers/nginx-plus-ingress.md [hunk 4]
```diff
@@ -61,8 +61,7 @@ In a Kubernetes environment, we use our Sumo Logic Kubernetes collection. You ca
...
-
+3. **Deployment of Sumologic Kubernetes Collection**: Ensure you have deployed the [Sumologic-Kubernetes-Collection](https://github.com/SumoLogic/sumologic-kubernetes-collection), to send the logs and metrics to Sumologic. For more information on deploying Sumologic-Kubernetes-Collection, [visit here](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/main/docs/installation.md). Once deployed, logs will automatically be picked up and sent by default. Prometheus will scrape the Nginx pods, based on the annotations set in Step 2, for the metrics. Logs and Metrics will automatically be sent to the respective [Sumo Logic Distribution for OpenTelemetry Collector](https://github.com/SumoLogic/sumologic-otel-collector) instances  which consistently tag your logs and metrics, then forward them to your Sumo Logic org.
```
Comment (kimsauce) [inline docs/integrations/web-servers/nginx-plus-ingress.md:64]:
```suggestion
3. **Deployment of Sumologic Kubernetes Collection**: Ensure you have deployed the [Sumologic-Kubernetes-Collection](https://github.com/SumoLogic/sumologic-kubernetes-collection), to send the logs and metrics to Sumologic. For more information on deploying Sumologic-Kubernetes-Collection, [visit here](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/main/docs/installation.md). Once deployed, logs will automatically be picked up and sent by default. Prometheus will scrape the Nginx pods, based on the annotations set in Step 2, for the metrics. Logs and Metrics will automatically be sent to the respective [Sumo Logic Distribution for OpenTelemetry Collector](https://github.com/SumoLogic/sumologic-otel-collector) instances, which consistently tag your logs and metrics, then forward them to your Sumo Logic org.
```

File: docs/integrations/web-servers/nginx.md [hunk 5]
```diff
@@ -88,11 +88,15 @@ Sumo Logic supports the collection of logs and metrics data from Nginx in both K
...
+The first service in the pipeline is Telegraf. Telegraf collects metrics from Nginx. Note that we’re running Telegraf in each pod we want to collect metrics from as a sidecar deployment: i.e. Telegraf runs in the same pod as the containers it monitors. Telegraf uses the [Nginx input plugin](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/nginx) to obtain metrics. For simplicity, the diagram doesn’t show the input plugins.
+The injection of the Telegraf sidecar container is done by the Telegraf Operator.
+Prometheus pulls metrics from Telegraf and sends them to [Sumo Logic Distribution for OpenTelemetry Collector](https://github.com/SumoLogic/sumologic-otel-collector) which enriches metadata and sends metrics to Sumo Logic.
```
Comment (kimsauce) [inline docs/integrations/web-servers/nginx.md:97]:
```suggestion
Prometheus pulls metrics from Telegraf and sends them to [Sumo Logic Distribution for OpenTelemetry Collector](https://github.com/SumoLogic/sumologic-otel-collector), which enriches metadata and sends metrics to Sumo Logic.
```

File: docs/integrations/web-servers/squid-proxy.md [hunk 6]
```diff
@@ -71,11 +71,15 @@ Click on the appropriate tabs below based on the environment where your Squid Pr
...
+The first service in the pipeline is Telegraf. Telegraf collects metrics from Squid Proxy. Note that we’re running Telegraf in each pod we want to collect metrics from as a sidecar deployment: i.e. Telegraf runs in the same pod as the containers it monitors. Telegraf uses the [SNMP input plugin](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/snmp) to obtain metrics. For simplicity, the diagram doesn’t show the input plugins.
+The injection of the Telegraf sidecar container is done by the Telegraf Operator.
+Prometheus pulls metrics from Telegraf and sends them to [Sumo Logic Distribution for OpenTelemetry Collector](https://github.com/SumoLogic/sumologic-otel-collector) which enriches metadata and sends metrics to Sumo Logic.
```
Comment (kimsauce) [inline docs/integrations/web-servers/squid-proxy.md:80]:
```suggestion
Prometheus pulls metrics from Telegraf and sends them to [Sumo Logic Distribution for OpenTelemetry Collector](https://github.com/SumoLogic/sumologic-otel-collector), which enriches metadata and sends metrics to Sumo Logic.
```

File: docs/integrations/web-servers/varnish.md [hunk 7]
```diff
@@ -107,11 +107,14 @@ Instructions below show how to configure Kubernetes and Non-Kubernetes environme
...
+The first service in the pipeline is Telegraf. Telegraf collects metrics from Varnish. Note that we’re running Telegraf in each pod we want to collect metrics from as a sidecar deployment, for example, Telegraf runs in the same pod as the containers it monitors. Telegraf uses the Varnish input plugin to obtain metrics. For simplicity, the diagram doesn’t show the input plugins.The injection of the Telegraf sidecar container is done by the Telegraf Operator.
+Prometheus pulls metrics from Telegraf and sends them to [Sumo Logic Distribution for OpenTelemetry Collector](https://github.com/SumoLogic/sumologic-otel-collector) which enriches metadata and sends metrics to Sumo Logic.
```
Comment (kimsauce) [inline docs/integrations/web-servers/varnish.md:115]:
```suggestion
Prometheus pulls metrics from Telegraf and sends them to [Sumo Logic Distribution for OpenTelemetry Collector](https://github.com/SumoLogic/sumologic-otel-collector), which enriches metadata and sends metrics to Sumo Logic.
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
Nice work! Just minor fixes.

File: docs/integrations/web-servers/haproxy.md (hunk 2 above)
Comment (kkujawa-sumo) [reply docs/integrations/web-servers/haproxy.md]: changed

File: docs/integrations/web-servers/nginx-plus-ingress.md (hunk 4 above)
Comment (kkujawa-sumo) [reply docs/integrations/web-servers/nginx-plus-ingress.md]: changed

**Review by kimsauce — APPROVED** [review]

**Pull request merged**